rules/framework.bzl

```diff
@@ -442,6 +464,53 @@ def _get_merged_swift_info(ctx, framework_files):
         swift_info_fields["modules"] = _copy_swiftmodule(ctx, framework_files)
     return swift_common.create_swift_info(**swift_info_fields)
 
+def _merge_root_infoplists(ctx):
+    if ctx.attr.infoplists == None or len(ctx.attr.infoplists) == 0:
+        return None
+
+    output_plist = ctx.actions.declare_file(
+        paths.join("%s-intermediates" % ctx.label.name, "Info.plist"),
+    )
+
+    bundle_name = ctx.attr.framework_name
+    current_apple_platform = transition_support.current_apple_platform(apple_fragment = ctx.fragments.apple, xcode_config = ctx.attr._xcode_config)
+    platform_type = str(current_apple_platform.platform.platform_type)
+    apple_toolchain_info = ctx.attr._toolchain[AppleSupportToolchainInfo]
+    rule_descriptor = rule_support.rule_descriptor(ctx)
+
+    resource_actions.merge_root_infoplists(
+        actions = ctx.actions,
+        bundle_name = bundle_name,
+        bundle_id = ctx.attr.bundle_id,
+        bundle_extension = ctx.attr.bundle_extension,
+        executable_name = bundle_name,
+        environment_plist = ctx.file.environment_plist,
+        input_plists = ctx.files.infoplists,
+        launch_storyboard = None,
+        output_plist = output_plist,
+        output_pkginfo = None,
+        platform_prerequisites = platform_support.platform_prerequisites(
+            apple_fragment = ctx.fragments.apple,
+            config_vars = ctx.var,
+            device_families = ["iphone", "ipad"],
+            explicit_minimum_os = None,
+            explicit_minimum_deployment_os = None,
```

Comment:
Needed for frameworks?

Reply:
minimum_deployment_os is derived from xcode config. explicit_minimum_deployment_os is to override the default setting. No need to do that for frameworks. We can add an attribute if necessary later.